backend/src/blocked/blocked.service.ts

```diff
@@ -85,4 +96,9 @@ export class BlockedService {
   /* 
   repository method
   */
+  async findBlockedByUserId(userId: number): Promise<number[]> {
```

Comment:
이거 사소하지만 이름 변경 가능할까요?
```suggestion
  async findBlockedIdByUserId(userId: number): Promise<number[]> {
```

Reply:
아ㅏ 인정입니다...ㅎ..